code-examples/kafka-java-maven/src/main/java/org/example/ProducerExample.java

```diff
@@ -0,0 +1,23 @@
+package org.example;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.common.serialization.StringSerializer;
+
+public class ProducerExample {
+
+    public static void main(String[] args) {
+
+        //Creating producer properties
+        var properties= KafkaConfig.properties();
+        properties.setProperty(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
+        properties.setProperty(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class.getName());
+
+        KafkaProducer<String,String> producer= new KafkaProducer<String,String>(properties);
```

Comment:
We already use `var` in this example so I think reducing clutter here on line 17 will be good by using it there aswell

Reply:
You mean something like `var producer = ` ?

Reply:
Yes exactly